docs/_posts/2013-10-3-community-roundup-9.md

```diff
@@ -56,7 +56,7 @@ The video will be available soon on the [JSConf EU website](http://2013.jsconf.e
 
 [Todd Kennedy](http://blog.selfassembled.org/) working at [Cond&eacute; Nast](http://www.condenast.com/) implemented a wrapper on-top of [JSHint](http://www.jshint.com/) that first converts JSX files to JS.
 
-> A wrapper around JSHint to allow linting of files containg JSX syntax. Accepts glob patterns. Respects your local .jshintrc file and .gitignore to filter your glob patterns.
+> A wrapper around JSHint to allow linting of files containing JSX syntax. Accepts glob patterns. Respects your local .jshintrc file and .gitignore to filter your glob patterns.
```

Comment:
:+1:  Haha, that's true, but it's an old blog post that preserves the community's thoughts of the time.  Sounds almost like @toddself may be advocating for an even more aggressive change, like adding an `[[UPDATE]]` note of some sort (that would be a larger and orthogonal discussion).  

Assuming we decide to keep the post in our archives, would you prefer that we fix the typo or keep the incorrect spelling?